classes/plugins/PubObjectsExportPlugin.inc.php

```diff
@@ -248,6 +273,22 @@ function getDepositSuccessNotificationMessageKey() {
 		return 'plugins.importexport.common.register.success';
 	}
 
+	/**
+	 * Get the locale key used in the notification for
+	 * the successful export validation.
+	 */
+	function getValidationExportSuccessNotificationMessageKey() {
+		return 'plugins.importexport.common.validation.success';
```

Comment:
Why not just code these values directly where they're used rather than creating functions for this? The functions only appear to be used in one case.